kernel-version.mk

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2023 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+# this file is included by Makefile
```

Comment:
I would suggesting the following (with fixed indentation and some minor fixes):
```
# This file is included by the main Makefile.
# Two variables are used as input for the kernel build:
#   ZARCH - architecture (amd64, arm64)
#   PLATFORM - platform (generic, nvidia, rt, imx*) based on these two
#              variables we set kernel version, commit hash and branch
#              when new kernel is built and pushed to the corresponding
#              dockerhub
#   KERNEL_COMMIT_xxx variable must be manually updated
```

Reply:
ack